src/libs/fileDownload/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as ApiUtils from '@libs/ApiUtils';
+import tryResolveUrlFromApiRoot from '@libs/tryResolveUrlFromApiRoot';
+import * as Link from '@userActions/Link';
+import CONST from '@src/CONST';
+import * as FileUtils from './FileUtils';
+import type {FileDownload} from './types';
+
+/**
+ * Downloading attachment in web, desktop
```

Comment:
Can you update this comment to a full correct sentence please?

Reply:
Ping on this one @fabioh8010

Reply:
@fvlvte Please have a look on this.